charts/nginx-gateway-fabric/templates/deployment.yaml

```diff
@@ -129,6 +129,8 @@ spec:
         volumeMounts:
         - name: nginx-conf
           mountPath: /etc/nginx/conf.d
+        - name: nginx-stream-conf
```

Comment:
Just now noticing this volume is always generated even though we only use it when experimental features are enabled. Not sure how much it matters, though. @sjberman do you think it's worth it to only generate this when experimental features are enabled?

Reply:
Since everything else in the chart is gated behind a condition, this probably should be too.